backend/src/Service/CaptainFinancialSystem/CaptainFinancialSystemAccordingToCountOfOrdersService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Service\CaptainFinancialSystem;
+
+use App\AutoMapping;
+use App\Entity\CaptainFinancialSystemAccordingToCountOfOrdersEntity;
+use App\Manager\CaptainFinancialSystem\CaptainFinancialSystemAccordingToCountOfOrdersManager;
+use App\Response\CaptainFinancialSystem\CaptainFinancialSystemAccordingToCountOfOrdersResponse;
+
+class CaptainFinancialSystemAccordingToCountOfOrdersService
+{
+    private CaptainFinancialSystemAccordingToCountOfOrdersManager $CaptainFinancialSystemAccordingToCountOfOrdersManager;
```

Comment:
Alter the first letter of the var name from capital to small letter, and check all places in which it is being used.